test/integration/02b-onchain-receive.spec.ts

```diff
@@ -183,32 +183,33 @@ it("createsBitcoindHotWallet", async () => {
 })
 
 it("user0IsCreditedForOnChainTransaction", async () => {
-  //   await bitcoind_onchain_funding({ walletDestination: walletUser0 })
-  await lnd_onchain_funding({ walletDestination: walletUser0 })
+  await bitcoind_onchain_funding({ walletDestination: walletUser0 })
+  //   await lnd_onchain_funding({ walletDestination: walletUser0 })
 })
 
 it("user11IsCreditedForOnChainSendAllTransaction", async () => {
   /// TODO? add sendAll tests in which the user has more than the limit?
   const level1WithdrawalLimit = yamlConfig.limits.withdrawal.level["1"] // sats
   amount_BTC = sat2btc(level1WithdrawalLimit)
   walletUser11 = await getUserWallet(11)
-  //   await bitcoind_onchain_funding({ walletDestination: walletUser11 })
-  await lnd_onchain_funding({ walletDestination: walletUser11 })
+  await bitcoind_onchain_funding({ walletDestination: walletUser11 })
+  //   await lnd_onchain_funding({ walletDestination: walletUser11 })
 })
 
 it("user12IsCreditedForOnChainOnUsSendAllTransaction", async () => {
   const level1OnUsLimit = yamlConfig.limits.onUs.level["1"] // sats
   amount_BTC = sat2btc(level1OnUsLimit)
   walletUser12 = await getUserWallet(12)
-  //   await bitcoind_onchain_funding({ walletDestination: walletUser12 })
-  await lnd_onchain_funding({ walletDestination: walletUser12 })
+  await bitcoind_onchain_funding({ walletDestination: walletUser12 })
+  //   await lnd_onchain_funding({ walletDestination: walletUser12 })
 })
 
 it("fundingFunderWithOnchainTxFromBitcoind", async () => {
-  //   await bitcoind_onchain_funding({ walletDestination: funderWallet })
-  await lnd_onchain_funding({ walletDestination: funderWallet })
+  await bitcoind_onchain_funding({ walletDestination: funderWallet })
+  //   await lnd_onchain_funding({ walletDestination: funderWallet })
 })
 
+// Now tests start to fail here (same reason: checkIsBalanced)
```

Comment:
This is what I plan to continue working on... but @nicolasburtey let me know if this is the best path or if I should focus on something else.